consultation_analyser/consultations/jinja2/consultations/responses/index.html

```diff
@@ -33,147 +45,153 @@ <h2 class="govuk-heading-m">Filters</h2>
+            <label class="govuk-label" for="theme">
+              Theme
+            </label>
+            <select class="govuk-select" id="theme" name="theme">
+              <option {% if applied_filters.theme == "All" %}selected{% endif %}>All</option>
+              {% for theme in themes %}
+                <option value="{{ theme.id }}" {% if applied_filters.theme|string == theme.id|string %}selected{% endif %}>{{ theme.short_description }}</option>
+              {% endfor %}
+              {#<option {% if applied_filters.theme == "No theme" %}selected{% endif %}>No theme</option>#}
+            </select>
           </div>
+         
+            {{ govukButton({
+              "text": "Apply filters",
+              "classes": "govuk-!-margin-bottom-2"
+            }) }}
+            <div class="govuk-!-padding-2">
+              <a class="govuk-body-s" href="?">Clear filters</a>
+            </div>
 
         </form>
       </div>
     </div>
 
-    <div class="govuk-grid-column-two-thirds">
-
-      <table class="govuk-table">
-        <caption class="govuk-table__caption govuk-table__caption--m">
-          <h2 class="govuk-heading-m govuk-!-margin-bottom-0">Responses</h2>
-          <span class="iai-table-subhead govuk-!-margin-top-1">(Showing <strong>{{ responses|length }}</strong> of <strong>{{ total_responses }}</strong> responses)</span>
-        </caption>
-        <thead class="govuk-table__head">
-          <tr class="govuk-table__row">
-            {% if question.multiple_choice_options %}
-              <th scope="col" class="govuk-table__header">Multiple choice</th>
-            {% endif %}
-            <th scope="col" class="govuk-table__header">Free text</th>
-          </tr>
-        </thead>
-        <tbody class="govuk-table__body">
-          {% for response in responses %}
-            <tr class="govuk-table__row govuk-body-s">
-              {% if response.multiple_choice %}
-                <td class="govuk-table__cell">
-                  {% for item in response.multiple_choice %}
-                    <strong>{{ item["question_text"] }}</strong> {{ ", ".join(item["options"]) }}<br />
-                  {% endfor %}
-                </td>
+    <div class="govuk-grid-column-three-quarters">
+      <div class="iai-panel">
+        <table class="govuk-table">
+          <caption class="govuk-table__caption govuk-table__caption--m govuk-!-margin-bottom-0">
+            <h2 class="govuk-body-s govuk-!-display-block govuk-!-font-weight-bold govuk-!-margin-bottom-0">Responses</h2>
+            <span class="iai-table-subhead govuk-body-s govuk-!-margin-top-1">(Showing <strong>{{ responses|length }}</strong> of <strong>{{ total_responses }}</strong> responses)</span>
+          </caption>
+          <thead class="govuk-table__head">
+            <tr class="govuk-table__row">
+              {% if question.multiple_choice_options %}
+                <th scope="col" class="govuk-table__header">Question</th>
```

Comment:
NB we can have multiple multiple-choice parts to a single question, hence `{% for item in response.multiple_choice %}` above — this might need another look

Reply:
We still have the `{% for item in response.multiple_choice %}`. But I think that's different to what you're saying. I can't see any difference in the for loops between this branch and what is on main - so is it also missing on main?